### PR TITLE
Cleanups of playbook syntax

### PR DESCRIPTION
**Pull request by j2sol**
These are backwards compatible cleanups I found when testing Ansible
2.0. 2.0 will warn about a lot of these, or fail on some of these.

### CODE REVIEW

File: roles/openstack-setup/tasks/networks.yml [hunk 1]
```diff
@@ -24,7 +24,6 @@
     ip_version: "{{ item.ip_version }}"
     ipv6_ra_mode: "{{ item.ipv6_ra_mode | default(omit) }}"
     ipv6_address_mode: "{{ item.ipv6_ra_mode | default(omit) }}"
-    ip_version: "{{ item.ip_version }}"
```
Comment (j2sol) [inline roles/openstack-setup/tasks/networks.yml]: ip_version is defined up on line 24 already, just removing a dupe.

**Comment by j2sol** [comment]
retest

File: playbooks/ci-full/tasks/create.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (masteinhauser) [inline playbooks/ci-full/tasks/create.retry]: why keep a retry file around?

File: playbooks/tests/tasks/main.retry [hunk 3]
```diff
@@ -0,0 +1 @@
+173.247.105.11
```
Comment (masteinhauser) [inline playbooks/tests/tasks/main.retry]: same, no need for retry files. Should add to the `.gitignore`

File: roles/apt-repos/tasks/main.yml [hunk 4]
```diff
@@ -3,13 +3,13 @@
   apt_key:
     url: "{{ item.key_url }}"
     validate_certs: "{{ item.validate_certs|default(omit) }}"
-  with_items: repos
+  with_items: "{{ repos|default([]) }}"
```
Comment (masteinhauser) [inline roles/apt-repos/tasks/main.yml]: If we have this defined in `apt-repos/defaults/main.yml` do we need it added on every usage? Or are you trying to standardize syntax?

File: playbooks/ci-full/tasks/create.retry (hunk 2 above)
Comment (j2sol) [inline playbooks/ci-full/tasks/create.retry]: Derp no idea why the retry files crept in. I was being so good about interactive additions to the commit.

File: roles/apt-repos/tasks/main.yml (hunk 4 above)
Comment (j2sol) [inline roles/apt-repos/tasks/main.yml]: We could define it there, but also yes, standardizing the syntax. Ansible 2.x will warn about A) raw variables in a with_items, and 2) it warns that the silent skipping of these tasks when the variable isn't defined is going away, and eventually they'll be errors. SO I was just going through and protecting every one, I didn't think of adding a default in the role itself.

Comment (masteinhauser) [inline roles/apt-repos/tasks/main.yml]: Both Ansible 2.x changes seem like the right direction, but I still feel like a role default makes more sense here and is more "ansibleithic" ?

Comment (j2sol) [inline roles/apt-repos/tasks/main.yml]: Yup, I'm testing that version now. looks good, so new push coming in a minute.

**Comment by j2sol** [comment]
retest

**Comment by nirajdp76** [comment]
LGTM

**Pull request merged**